docs/Makefile

```diff
@@ -13,7 +13,7 @@ help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
 figures:
-	python source/scripts/build_activation_images.py
+	python3 source/scripts/build_activation_images.py
```

Comment:
Is it possible to find out the version of Python installed for this?

Reply:
Oh, that's awkward. So this is really Python 3 only? It is a basically safe assumption to assume python 3 is installed with a `python3` symlink, but it would be even better if this wasn't a Python 3 only thing.

Reply:
I built `torch` with Python3. This is what I did: `sudo python3 setup.py build develop`.

Maybe because I am explicitly specifying `python3` is causing this to happen.

Reply:
It's not Python 3 only, it's "the python you used to build PyTorch" only. I can see two possibilities:
- make the Python command to use a Makefile variable (that can be overriden using an env variable). We already use this strategy in `test/run_test.sh`.
- Do `python <script_path> || python3 <script_path>` to handle failures automatically. A bit non-obivous, so I'd lean towards the first solution